src/components/assets/Erc20Currency.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div>
+    <div class="border--separator" />
+    <div class="rows">
+      <div class="row row--details">
+        <div class="row__left">
+          <div class="column--currency">
+            <img :src="token.image" :alt="token.name" class="token-logo" />
+            <div class="column--ticker">
+              <span class="text--title">{{ token.symbol }}</span>
+              <span class="text--label">{{ token.name }}</span>
+            </div>
+          </div>
+        </div>
+        <div class="row__right">
+          <div class="column column--balance">
+            <div class="column__box">
+              <div class="text--accent">
+                <span>{{ $n(Number(token.userBalance)) }} {{ token.symbol }}</span>
+              </div>
+              <div class="text--label">
+                <span>{{ $n(Number(token.userBalanceUsd)) }} {{ $t('usd') }}</span>
+              </div>
+            </div>
+          </div>
+          <div class="column--asset-buttons column--buttons--multi">
+            <button
+              class="btn btn--sm"
+              @click="
+                handleModalTransfer({
+                  isOpen: true,
+                  currency: token,
+                })
+              "
+            >
+              {{ $t('assets.transfer') }}
+            </button>
+            <div v-if="token.isWrappedToken">
+              <a :href="token.wrapUrl" target="_blank" rel="noopener noreferrer">
+                <button class="btn btn--sm">{{ $t('assets.wrap') }}</button>
+              </a>
+            </div>
+            <a :href="explorerLink" target="_blank" rel="noopener noreferrer" class="screen--xl">
+              <button class="btn btn--sm">
+                {{ $t('assets.explorer') }}
```

Comment:
How about an icon button like the one on the account's explorer view?

Reply:
an icon makes sense to me. It's nice to have more contents:) 

Love the wrap too!  I understand we won't have this in mobile. In the future I'd add three dot icon in mobile view for those extra contents.

Reply:
Fixed!